components/p3a/metric_log_store.cc

```diff
@@ -277,7 +274,7 @@ void MetricLogStore::DiscardStagedLog(std::string_view reason) {
 void MetricLogStore::MarkStagedLogAsSent() {}
 
 void MetricLogStore::TrimAndPersistUnsentLogs(bool overwrite_in_memory_store) {
```

Comment:
not for this pr, but we can refactor out obsolete inheritance and drop unused methods